itest/addrs_test.go

```diff
@@ -931,6 +932,14 @@ func withReceiverAddresses(addrs ...*taprpc.Addr) sendOption {
 	}
 }
 
+// withSkipProofCourierPingCheck is an option to skip the proof courier ping
+// check. This is useful for testing purposes.
+func withSkipProofCourierPingCheck(skipCheck bool) sendOption {
```

Comment:
nit: since the default value of a bool is `false`, I think we don't need a parameter here and can just set this to `true`. Since I think we'd never want to turn it on and later off again?